source/lib-intro.tex

```diff
@@ -476,7 +476,7 @@
 appropriate):\footnote{To save space, items that do not apply to a function are omitted.
 For example, if a function does not specify any
 further
-preconditions, there will be no ``Requires'' paragraph.}
+preconditions, there will be no \requires paragraph.}
```

Comment:
"Element" is the term used when they are defined, and the term LWG has been using for them recently.